docs/quick-start.md

```diff
@@ -733,32 +733,31 @@ Make sure to use the same Host header you defined in your `VMAuth` config, or th
 
 ## Next steps
 
-That's it. We obtained a monitoring cluster corresponding to the target topology:
+Now that you have a working setup with [storage](https://docs.victoriametrics.com/operator/quick-start/#storage), [scraping](https://docs.victoriametrics.com/operator/quick-start/#scraping), [alerting](https://docs.victoriametrics.com/operator/quick-start/#alerting), and secure [access](https://docs.victoriametrics.com/operator/quick-start/#access),
+and a solid understanding of how everything fits together,
+you’re ready to take the next steps with VictoriaMetrics on Kubernetes.
 
-![Cluster Scheme](quick-start_cluster-scheme.webp)
+You might want to try a different way to install it.
+The [victoria-metrics-k8s-stack Helm chart](https://docs.victoriametrics.com/operator/setup/#installing-by-helm-charts) is a good choice if you want everything in one package — 
+VictoriaMetrics, Grafana, scraper and data sources — all preconfigured to work together.
 
-You have a full-stack monitoring cluster with VictoriaMetrics Operator.
+If you’re moving from Prometheus, VictoriaMetrics [makes the migration easy](https://docs.victoriametrics.com/operator/migration/).
+You can keep your current scrape configs and alert rules, and move your data with just a few changes.
 
-You can find information about these and other resources of operator on the [Custom resources page](https://docs.victoriametrics.com/operator/resources/).
+As you continue to explore, you might find the [resources](https://docs.victoriametrics.com/operator/resources/) and [their configuration reference](https://docs.victoriametrics.com/operator/api/) pages helpful.
 
-In addition, check out other sections of the documentation for VictoriaMetrics Operator:
+If you're expecting more traffic or need better uptime, consider switching from [VMSingle](https://docs.victoriametrics.com/operator/resources/vmsingle/) to [VMCluster](https://docs.victoriametrics.com/operator/resources/vmcluster/).
```

Comment:
I'd make a last attempt to agitate for using single-node.

```
While [VMSingle](https://docs.victoriametrics.com/operator/resources/vmsingle/) has great vertical scalability capabilities and efficiency, [VMCluster](https://docs.victoriametrics.com/operator/resources/vmcluster/) allows scaling beyond capabilities of a single physical instance.
```

Reply:
Agree, I rewrote the text to stress that vmsingle is the recommended way.

PTAL

Reply:
Thanks! It is much better than what I suggested!